### DISCUSSION TITLE
Messages?

### DISCUSSION THREAD

**Discussion by zarkones** [community/community#17888] [Discussions]
It would be convenient and will bring the community closer together,
if we could exchange messages and talk to each other on the platform.
Thanks for listening, what do you think about chat functionality?

**Comment by MrKrishnaAgarwal** [comment]
It's a great Idea but other social media platforms like: Twitter & LinkedIn provide it.
So I think it's better there is no chat as GitHub is for Open-source

**Reply by zarkones** [reply]
@MrKrishnaAgarwal But those platforms don't connect the group of people GitHub does, and in a way it does it.

**Reply by atesin** [reply]
i don't have any social network account and won't create any, don't like them...  and maybe the people i want to talk doesn't have any either... 
or have, but dont want to share it... or have and shares it but i don't know it... or i have a social network account, he/she have too, want to share it, i know it, but in his/her social networks just want to talk about shopping shoes and eat sandwiches and travels and not any about github...

i would like private messages on github, through tools provided by github, with people of github that i am SURE is in github and can find on github, to talk (sync) about github... you know, ACTUAL software projects and stuff (not shopping, travel or gossip).... it would add a little ease and productiveness

**Comment by ale-lit** [comment]
And I love this idea. Also sometimes there is not enough private messaging function.

**Reply by zarkones** [reply]
@ale-lit By private, you mean the one providing anonymity by obscuring the metadata?

**Comment by d-coder111** [comment]
> It would be convenient and will bring the community closer together, if we could exchange messages and talk to each other on the platform. Thanks for listening, what do you think about chat functionality?

In my opinion that would be really cool for people to connect well.

**Comment by syspone** [comment]
Concordo plenamente, me parece que o GitHub sem essa relação de intercambio aparentemente parece engessado. Da uma senção que você está só nesse vasto mundo dos códigos.

**Comment by jfullstackdev** [comment]
is this Discussion feature not enough for us to communicate ? but a private message is a different thing, why would we be needing private message in this platform when it's always team collaboration ? 

for teams collaborating privately, Discussion can still be enabled

**Reply by atesin** [reply]
agree... but additionally, being able to private message ANYBODY on github regardless some repo or not, won't hurt... actually would be easier and provide better communication

if don't like it just don't use it... but don't force everybody to think or do the same as you

i HAD the need to communicate with some people before, to discuss ideas about NEW projects

**Reply by carbon-starlight** [reply]
Well, I just had a situation when I was talking to a person in an issue discussion, and a maintainer asked to move our discussion away as off-topic. This person does not have contacts in bio. PM feature would be useful.

**Comment by jfullstackdev** [comment]
I am not against this private messaging but as for now, as a workaround that will require extra steps to achieve the same thing, and I'm very sure if the other user does not want, he / she will not proceed,

create a private repo, add the other user as a collaborator, start messaging maybe in Issue or Discussion tab

**Reply by atesin** [reply]
yes...  we have to do that sort of _tricks_ on github **because** there is no native private message function...  anyway is a little annoying and does not guarantee an easy and agile communication (i.e. no notificacions, no easy access, etc)

**Reply by jfullstackdev** [reply]
> agile communication (i.e. no notificacions, no easy access, etc)

how come ? it's being notified when they reply, like my notification 

![image](https://user-images.githubusercontent.com/47092464/182010527-5f188b7f-659e-4905-9ee3-c0c53f0c4292.png)

once you set this up, it's done, bookmark the conversation and you're one click away

**Reply by atesin** [reply]
you are right....  but as private messages don't exist currently, what would you have to do if this discussion wouldn't exist previously, and you **want/need** to communicate with me???...

that is all we are talking about, is annoying

**Reply by jfullstackdev** [reply]
I'm just presenting the workaround while it's not yet implemented, so once it's set, you will be seeing that at your notif, like the one I posted above

**Reply by atesin** [reply]
"private message"...  you know, like other online tools, we are not reinventing the wheel

open some user profile > there is some "send private message to this user" button > you click it and a text area appears  > you write some message and click the "send" button

the user receives a mail notification of "new private message" with possibly a direct link to his/her profile inbox page > he/she opens his profile inbox and see unread messages > can respond to messages in a textarea below the message, and send the response starting a private conversation this way... 

"private conversation", because anyone else but sender and receiver can see it through his/her "profile inbox"  ... however could also possibly invite more people to the conversation, would be private for "them"

you see there is no repository or issue or patch req involved, or any previous step or requisite or even any message to respond... just send a "new" pm directly from the user profile page (or picture)

**Comment by upintheairsheep** [comment]
Maybe add "user issues"?
